documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -6,20 +9,2563 @@
+
+[source,json,index=0]
+----
+{
+ "schema": { <1>
+    "type": "struct",
+    "name": "mysql-server-1.inventory.customers.Key", <2>
+    "optional": false, <3>
+    "fields": [ <4>
+      {
+        "field": "id",
+        "type": "int32",
+        "optional": false
+      }
+    ]
+  },
+ "payload": { <5>
+    "id": 1001
+  }
+}
+----
+
+.Description of change event key
+[cols="1,2,7",options="header"]
+|===
+|Item |Field name |Description
+
+|1
+|`schema`
+|The schema portion of the key specifies a Kafka Connect schema that describes what is in the key's `payload` portion. 
+
+|2
+|`mysql-server-1.inventory.customers.Key`
+a|Name of the schema that defines the structure of the key's payload. This schema describes the structure of the primary key for the table that was changed. Key schema names have the format _connector-name_._database-name_._table-name_.`Key`. In this example: + 
+
+* `mysql-server-1` is the name of the connector that generated this event. + 
+* `inventory` is the database that contains the table that was changed. +
+* `customers` is the table that was updated.
+
+|3
+|`optional`
+|Indicates whether the event key must contain a value in its `payload` field. In this example, a value in the key's payload is required. A value in the key's payload field is optional when a table does not have a primary key.
+
+|4
+|`fields` 
+|Specifies each field that is expected in the `payload`, including each field's name, type, and whether it is required.
+
+|5
+|`payload`
+|Contains the key for the row for which this change event was generated. In this example, the key, contains a single `id` field whose value is `1001`.
+
+|===
+
+// Type: concept
+// ModuleID: about-values-in-debezium-mysql-change-events
+// Title: About values in {prodname} MySQL change events
+[[mysql-change-event-values]]
+=== Change event values
+
+The value in a change event is a bit more complicated than the key. Like the key, the value has a `schema` section and a `payload` section. The `schema` section contains the schema that describes the `Envelope` structure of the `payload` section, including its nested fields. Change events for operations that create, update or delete data all have a value payload with an envelope structure. 
+
+Consider the same sample table that was used to show an example of a change event key: 
+
+[source,sql]
+----
+CREATE TABLE customers (
+  id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL UNIQUE KEY
+) AUTO_INCREMENT=1001;
+----
+
+The value portion of a change event for a change to this table is described for: 
+
+* <<mysql-create-events,_create_ events>>
```

Comment:
I'm of the opinion that now that we've added Tombstone and PK updates options here, should we consider formatting these entries despite how the headings are may formatted?  To me this goes back to having camel-case elements and upper-case elements when we discussed `HSTORE` and `TIMESTAMP` in the PG docs.

Reply:
Sorry. I don't understand what you want to see in the doc.
The way it is now, the user will see these links: 

_create_ events
_update_ events
Primary key updates
_delete_ events
Tombstone events

How should this change?

Reply:
I wonder if it would look more uniform if the text here was

Create events
Update events
Primary key updates
Delete events
Tombstone events

Reply:
Agreed, the italic setting looks a bit odd when looking at it in the navigation menu.

Reply:
I feel strongly that the headings need to immediately convey, as much as possible, that "create", "delete", and "update" are being used as adjectives and not as imperative verbs. While yes, the headings you suggest, Chris, are uniform, they are less helpful to a user who is trying to learn. 

We already made the update I am suggesting in the doc for Db2 and for PostgreSQL. Whatever we decide here, will need to be applied to the doc for those connectors, and to the doc for the other connectors as we update their doc. You can see how it looks in the navigation panel now: 
https://debezium.io/documentation/reference/1.2/connectors/postgresql.html#postgresql-events
https://debezium.io/documentation/reference/1.2/connectors/db2.html#db2-events 

Please reconsider leaving it the way I did it. 
Alternatively, I would be okay (though not as happy) with:

CREATE events
UPDATE events
Primary key updates
DELETE events
Tombstone events